uptime-kuma/deploy.yaml

```diff
@@ -1,37 +1,46 @@
 ---
-version: "2.0"
-
+version: '2.0'
 services:
-  uptime:
-    image: louislam/uptime-kuma:1.18.0
+  uptime-kuma-persistent:
+    image: 'louislam/uptime-kuma:1.23.1'
     expose:
       - port: 3001
         as: 80
         to:
           - global: true
+    params:
+      storage:
+        data:
+          mount: /app/data
+          readOnly: false
 profiles:
   compute:
-    uptime:
+    uptime-kuma-persistent:
       resources:
         cpu:
           units: 0.5
         memory:
-          size: 128Mi
+          size: 512Mb

```

Comment:
any reason to use `Mb` instead of `Mi`

Reply:
Mb (SI / decimal) is much more human readable, compared to Mi which is (binary / power of 2)

IE output of 
curl -sk https://dcnorse.ddns.net:8443/status | jq

 {
          "cpu": 2000,
          "gpu": 0,
          "memory": 512000000,
          "storage_ephemeral": 536870912
        }

memory is in the above example defined by Mb while storage is in Mi.
maybe its just my personal preference, but it just makes life a little easier to work in decimal.
which i guess is why the storage manufactures also work in Mb,Gb,Tb...

Reply:
I understand the appealing it might be to use more readable numbers but ram and storage is inherently binary in nature. Using SI units in such contexts can lead to confusion and imprecise measurements. There is some precision losing, in this case a lot more, and I would image in this industry for Mi to mean more than Mb. 

Not for me to decide but I would just follow what the others are doing and use `Mi` instead `Mb`

Reply:
actually storage is not done in binary anymore, and hasn't been for many years.

quote from wiki
"By the 1970s, the total capacity of HDDs was given by manufacturers using [SI](https://en.wikipedia.org/wiki/SI) decimal prefixes such as"
from
https://en.wikipedia.org/wiki/Hard_disk_drive

i'm going to risk it, lol then maybe one day ill figure out a good reason not to use it.

and really isn't it the other way around.... its the power of 2 based binary which is prone for people to make errors in.
the SI decimal system is a standard for a reason...

also for storage its the fundamental nature how the storage calculated from the factory...
if you buy a 6TB storage media, you are not getting 6TiB.... 

and does it really matter.... it requires terabytes before the deviation is beyond a few %
which is something we almost never see on akash.

Reply:
I am not saying you are not wrong, I agree that is easier for a "common" to read Mb instead the binary version, but we are not really "common". In this context and in similarity with what is done on literally everywhere in this repo I still think binary is preferable. For precision and accuracy, I would rather use the binary annotation than latter notice that I am missing some "MB" just because the annotation was wrong. 

I would rather leave to the UI to make it more pleasant for "commoners" than here on a high level repo. Take cloudmos for example: 

![image](https://github.com/akash-network/awesome-akash/assets/3440849/94158d39-ab53-403d-a96c-7bf7890876ad)

Reply:
That said is not for me to decide - So I just leave here my opinion :) 

honestly, feels like you are spending more time going against something that has been done in the repo, fruitless. You should open an "issue" and present your points if you feel strongly about it, reducing the friction on your PR and get it merged easier with the standards of the repo.